servicetalk-opentracing-zipkin-publisher/src/main/java/io/servicetalk/opentracing/zipkin/publisher/reporter/HttpReporter.java

```diff
@@ -69,7 +69,7 @@
     static final String V1_PATH = "/api/v1/spans";
     static final String V2_PATH = "/api/v2/spans";
     static final CharSequence THRIFT_CONTENT_TYPE = newAsciiString("application/x-thrift");
-    static final CharSequence PROTO_CONTENT_TYPE = newAsciiString("application/protobuf");
+    static final CharSequence PROTO_CONTENT_TYPE = newAsciiString("application/x-protobuf");
```

Comment:
Consider enhancing `HttpReporterTest#verifyRequest` (btw, while you are there, please make that method `static`) to also validate content-type.